detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/Finding2.kt

```diff
@@ -3,13 +3,18 @@ package io.gitlab.arturbosch.detekt.api
 /**
  * Represents a problem detected by detekt on the source code
  *
- * A finding has an issue (information about the rule that detected the problem), a severity and a source code position
+ * A finding has the information about the rule that detected the problem, a severity and a source code position
```

Comment:
```suggestion
 * A finding has information about the rule that detected the problem, a severity and a source code position
```

Reading this now it might make sense to remove the reference to "source code position" since it doesn't hold that directly, instead it's a list of entities that caused the finding.